internal/driver/commands.go

```diff
@@ -336,25 +336,26 @@ func listHelp(c string, redirect bool) string {
 }
 
 // browsers returns a list of commands to attempt for web visualization.
-// Commands which definitely will open a browser are prioritized over other
-// commands like xdg-open,  which may not open the javascript embedded SVG
-// files produced by the -web command in a browser.
 func browsers() []string {
 	var cmds []string
 	if userBrowser := os.Getenv("BROWSER"); userBrowser != "" {
 		cmds = append(cmds, userBrowser)
 	}
-	cmds = append(cmds, []string{"chrome", "google-chrome", "chromium", "firefox"}...)
 	switch runtime.GOOS {
 	case "darwin":
 		cmds = append(cmds, "/usr/bin/open")
 	case "windows":
 		cmds = append(cmds, "cmd /c start")
 	default:
+		// Commands opening browsers are prioritized over xdg-open, so browser()
+		// command can be used on linux to open the .svg file generated by the -web
+		// command (the .svg file includes embedded javascript so is best viewed in
+		// a browser).
+		cmds = append(cmds, []string{"chrome", "google-chrome", "chromium", "firefox", "sensible-browser"}...)
 		if os.Getenv("DISPLAY") != "" {
+			// xdg-open is only for use in a desktop environment.
```

Comment:
FWIW, I don't think there is much sense in opening Chrome when DISPLAY is not set either, but we can save it for a future change.